pkg/oc/cli/cli.go

```diff
@@ -330,14 +352,14 @@ func CommandFor(basename string) *cobra.Command {
 
 	switch basename {
 	case "kubectl":
-		cmd = kubecmd.NewKubectlCommand(in, out, errout)
+		cmd = kubecmd.NewDefaultKubectlCommand()
 	case "openshift-deploy":
 		cmd = deployer.NewCommandDeployer(basename)
 	case "openshift-recycle":
 		cmd = recycle.NewCommandRecycle(basename, out)
 	default:
 		shimKubectlForOc()
-		cmd = NewCommandCLI("oc", "oc", in, out, errout)
```

Comment:
While you're at it rename this to be `NewDefaultOcCommand` and the next to `NewOcCommand`